runelite-client/src/main/java/net/runelite/client/plugins/bankvalue/BankTitle.java

```diff
@@ -92,12 +92,26 @@ void update(long gePrice, long haPrice)
 
 		if (config.showGE() && gePrice != 0)
 		{
-			strCurrentTab += " (EX: " + StackFormatter.quantityToStackSize(gePrice) + ")";
+			if (config.showExact())
+			{
+				strCurrentTab += " (EX: " + StackFormatter.formatNumber(gePrice) + ")";
```

Comment:
Since the `(EX: ... )` parts of the string will always be added, they should be added outside of the if statement.